### PR TITLE
Switch en-US FTL file to new syntax

### PR DESCRIPTION
**Pull request by flodolo**
* `#` for comment sigil
* no more sections,`##` for section comments
* equal sign on empty value
* also used spaces around equal sign consistently

CC @pike

### CODE REVIEW

File: locales/en-US/app.ftl [hunk 1]
```diff
@@ -1,23 +1,20 @@
 siteName = Firefox Test Pilot
 
-// Page titles, put in the <title> HTML tag.
-[[pageTitle]]
+## Page titles, put in the <title> HTML tag.
```
Comment (Pike) [inline locales/en-US/app.ftl]: drive-by nit, put-put-put. `shown as title of the html page` is less techy, and doesn't sound like you should add <title> around your strings.

File: locales/en-US/app.ftl [hunk 2]
```diff
@@ -1,23 +1,20 @@
...
 pageTitleDefault = Firefox Test Pilot
 pageTitleLandingPage = Firefox Test Pilot
 pageTitleExperimentListPage = Firefox Test Pilot - Experiments
 pageTitleExperiment = Firefox Test Pilot - {$title}
```
Comment (Pike) [inline locales/en-US/app.ftl]: Should we add a Term for Firefox Test Pilot?

File: locales/en-US/app.ftl [hunk 3]
```diff
@@ -26,27 +23,23 @@ menuFileIssue = File an Issue
...
+## Related to the installation of the Test Pilot add-on.
 landingInstallButton = Install the Test Pilot Add-on
 landingInstallingButton = Installing…
 landingEnablingButton = Enabling…
 
-// Related to a one click to install test pilot and an experiment.
-[[oneClickInstall]]
+## Related to a one click to install test pilot and an experiment.
 oneClickInstallMinorCta = Install Test Pilot &amp;
 oneClickInstallMajorCta = Enable {$title}
```
Comment (Pike) [inline locales/en-US/app.ftl]: `title` could use a descriptive comment.

File: locales/en-US/app.ftl [hunk 4]
```diff
@@ -55,41 +48,35 @@ landingDownloadFirefoxTitle = Firefox
...
-[[onboarding]]
+## Shown after the user installs the Test Pilot add-on.
 onboardingMessage = We put an icon in your toolbar so you can always find Test Pilot.
 
-// Error message pages.
-[[error]]
+## Error message pages.
 errorHeading = Whoops!
 errorMessage = Looks like we broke something. <br> Maybe try again later.
 notFoundHeader = Four Oh Four!
```
Comment (Pike) [inline locales/en-US/app.ftl]: Should this have a comment? Looking at existing translations, it might.

File: locales/en-US/app.ftl (hunk 2 above)
Comment (flodolo) [reply locales/en-US/app.ftl]: It might make sense, but in case it should be in a different issue/discussion, given how many translations we would lose.

**Comment by flodolo** [comment]
Thanks, I think I've fixed all the comments.

**Review by chuckharmston — APPROVED** [review]
